paasta_tools/cli/cmds/spark_run.py

```diff
@@ -326,6 +353,54 @@ def get_spark_conf_str(
     volumes,
 ):
     spark_conf = list()
+
+    base_args = {
+        'spark.cores.max': '4',
+        'spark.executor.cores': '2',
+        'spark.executor.memory': '4g',
+    }
+
+    if args.spark_args:
```

Comment:
nitpick, but it might be nice to put this into a method called something like `parse_spark_args`?